CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 ### Merged
 
-- Align Ready method's behaviour   [`#522`](https://github.com/rudderlabs/rudder-sdk-js/pull/522)
+- Align Ready method's behaviour [`#522`](https://github.com/rudderlabs/rudder-sdk-js/pull/522)
```

Comment:
Let's revert the changes from this file.

Reply:
Addressed